Fabric.Identity.API/scripts/Register.ps1

```diff
@@ -0,0 +1,570 @@
+#
+# Register.ps1
+#
+function Get-IdentityServiceUrl()
+{
+    return "https://$env:computername.$($env:userdnsdomain.tolower())/Identity"
+}
+
+function Get-AuthorizationServiceUrl()
+{
+    return "https://$env:computername.$($env:userdnsdomain.tolower())/Authorization"
+}
+
+function Get-DiscoveryServiceUrl()
+{
+    return "https://$env:computername.$($env:userdnsdomain.tolower())/DiscoveryService"
+}
+
+function Get-ApplicationUrl($serviceName, $discoveryServiceUrl){
+    $discoveryRequest = "$discoveryServiceUrl/v1/Services?`$filter=ServiceName eq '$serviceName'&`$select=ServiceUrl&`$orderby=Version desc"
+    $discoveryResponse = Invoke-RestMethod -Method Get -Uri $discoveryRequest -UseDefaultCredentials
+    $serviceUrl = $discoveryResponse.value.ServiceUrl
+    if([string]::IsNullOrWhiteSpace($serviceUrl)){
+        Write-Error "The service $serviceName is not registered with the Discovery service. Make sure that the service is registered w/ Discovery service before proceeding. Halting installation." -ErrorAction Stop
+    }
+    return $serviceUrl
+}
+
+function Invoke-Get($url, $accessToken)
+{
+    $headers = @{"Accept" = "application/json"}
+    if($accessToken){
+        $headers.Add("Authorization", "Bearer $accessToken")
+    }
+        
+    $getResponse = Invoke-RestMethod -Method Get -Uri $url -Headers $headers
+    return $getResponse
+}
+
```

Comment:
Yes, I believe we do, since we are posting/putting json we need to tell the server we are posting json. By default the Invoke-RestMethod sets the Content-Type to `application/x-www-form-urlencoded`, and the way you change that is by setting a parameter when invoking the request, check line 50 for an example.

Reply:
Oh ok - I missed that. I thought we weren't setting it anywhere.